docs/src/main/sphinx/sql/create-table-as.md

```diff
@@ -3,7 +3,7 @@
 ## Synopsis
 
 ```text
-CREATE TABLE [ IF NOT EXISTS ] table_name [ ( column_alias, ... ) ]
+CREATE [ OR REPLACE ] TABLE [ IF NOT EXISTS ] table_name [ ( column_alias, ... ) ]
 [ COMMENT table_comment ]
```

Comment:
Add explanation for the purpose of `OR REPLACE`.
Add note saying that `OR REPLACE` and `IF NOT EXISTS` can't be used together.